### PR TITLE
feat(www): Re-Organize tags on starters.yml

### PR DESCRIPTION
**Pull request by LekoArts**
- Added "Headless CMS" to entries appropriately
- Removed minor tags that create extra noise (Recompose, Prettier, Lint-Staged, TSLint, Tags) and removed them or put them into "Linting". Especially considering that almost all starters have ESLint and Prettier. "Linting" would mean for me that this starter is especially focused on that or has something fancy
- Removed invalid Styling tag

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -1075,7 +1084,7 @@
   description: Blog Template X Contentful, Twitter and Facebook style
   tags:
     - Blog
-    - Tags
```
Comment (pieh) [inline docs/starters.yml]: I think intention here is that starters has support for post tags?

Comment (LekoArts) [reply docs/starters.yml]: Mhhh, no one used that even though the starter has it. You could also call that categories, so what to choose here then?

Comment (pieh) [reply docs/starters.yml]: I probably miss-understood intention (I thought this was one of invalid tags). On that I have no opinion

**Comment by LekoArts** [comment]
@shannonbux ^

**Pull request merged**